hashintel/hash
# Load root `.env*` files in HASH frontend

## 🌟 What is the purpose of this PR?

This PR allows us to propagate env vars from `[root]/.env*` files down to Next.

## 🐾 Next steps

- Complete #175 

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1201095311341924/1201681664981918/f) _(internal)_

## ❓ How to test this?

1.  Add `console.log(process.env.NEXT_PUBLIC_XYZ)` in any app file inside `packages/hash/frontend/src`.
1.  Add `NEXT_PUBLIC_XYZ=42` to `.env` or `.env.local` in the repo root.
1.  Run the app without this change and observe that the value is `undefined`
1.  Check out this PR’s branch and repeat the above steps. You should see `"42"` instead of `undefined`.
